Resilio Connect API/JSMonitor/utils.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    findArrayDiff,
+};
+
+function findArrayDiff(a1, a2) {
+    let a = []
+    let diff = [];
+
+    for (var i = 0; i < a1.length; i++) {
+        a[a1[i]] = true;
+    }
+
+    for (var i = 0; i < a2.length; i++) {
+        if (a[a2[i]]) {
+            delete a[a2[i]];
+        } else {
+            a[a2[i]] = true;
+        }
+    }
+
+    for (var k in a) {
+        diff.push(Number(k));
+    }
+
+    return diff;
+}
```

Comment:
Could you please document this function?  Add 4-5 comments that explain the logic of this function.
It would also be good to use more meaningful variable names for 'a' and 'k'

Reply:
This was something I found off of StackOverflow. I am looking to see if I can create a less complex function that doesn't use 3 for loops.